### ISSUE TITLE
Add kosagi.list file to /etc/apt/sources.list.d instead of modfiying sources.list

### ISSUE THREAD

**Issue by fincham** [thesourcerer8/novena-debian-installer#2]
When the repo is added to sources.list it causes an error when kosagi-repo package is installed.

Installing the repo lines to /etcv/apt/sources.list.d/kosasgi.list would allow the kosagi-repo package to overwrite the file.

**Comment by thesourcerer8** [comment]
Yes, I would like to use the kosagi-repo package instead of adding the repo lines to sources.list , but I currently do not have an idea how I could install the package without adding the repository first. Any ideas?

**Comment by kop316** [comment]
This may be of use:

http://stackoverflow.com/questions/5410757/delete-a-line-containing-a-specific-string-using-sed

(I just noticed it now, or else I would have tried to include it in my updates)

**Comment by thesourcerer8** [comment]
Now I have another idea: I could package a copy of the kosagi-repo.deb file and install it during the installation from the local file
